main.py

```diff
@@ -170,14 +176,14 @@ async def invoke(self, ctx: StellaContext, **flags) -> None:
             if flags.pop("redirect_error", False):
                 raise exc
 
+    def sync_is_owner(self, user: discord.User) -> bool:
+        return user.id in self.user_ids
```

Comment:
```python
self.user_ids
``` 
came from where

Reply:
From here: https://github.com/Rapptz/discord.py/blob/45d498c1b76deaf3b394d17ccf56112fa691d160/discord/ext/commands/bot.py#L136
Refer to: https://github.com/Rapptz/discord.py/blob/45d498c1b76deaf3b394d17ccf56112fa691d160/discord/ext/commands/bot.py#L345

Actually, async is_owner is useless if you manually pass owner_ids to bot constructor. It won't update list from app info.

I fetch owners after bot init: https://github.com/Fogapod/pink/blob/0594793ecb79d465a4916c9c8764e73fb8bf9235/pink/bot.py#L175
Maybe do the same here? It won't hurt if bot processes a few messages with empty owner_ids or refetches owners on reconnect

Reply:
sir use your eyes, you're doing `self.user_ids`, not `self.owner_ids`

Reply:
So true